bin/bootstrap.sh

```diff
@@ -19,6 +19,7 @@ git commit -m 'Initial commit' -m 'Initialized from lisk-template: https://githu
 
 # Customize project using provided project name
 git grep -l lisk-template | xargs sed -i '' "s/lisk-template/$projectname/g"
+sed -i '' $'s/\t"version": .*/\t"version": "0.0.0",/' package.json
```

Comment:
What is this first empty string (`''`) for?

Reply:
Specifies no backup should be made of the original file despite editing in-place.

Reply:
OK. Isn't that the default behaviour (when no backup extension is specified) though?

Reply:
What would be the syntax in that case? I get the following if I leave it blank:
```
$ sed -i $'s/\t"version": .*/\t"version": "0.0.0XXX",/' package.json
sed: 1: "package.json": extra characters at the end of p command
```

Reply:
Looks like a difference between GNU sed and whatever ships with MacOS maybe? @Nazgolze FYI